.circleci/config.yml

```diff
@@ -1,27 +1,42 @@
-# Javascript Node CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-javascript/ for more details
-#
 version: 2
 jobs:
-  build:
+  test:
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:8.9
```

Comment:
why exacly?

Reply:
It's always preferable to specify exact version. Unless we want to always run latest.